cluster/saltbase/salt/kube-addons/initd

```diff
@@ -21,9 +21,6 @@ PIDFILE=/var/run/$NAME.pid
 SCRIPTNAME=/etc/init.d/$NAME
 KUBE_ADDONS_SH=/etc/kubernetes/kube-addons.sh
 
-# Read configuration variable file if it is present
```

Comment:
This is the salt-generated default file that I deleted above. Because I switched it to use the kubernetes service DNS address instead of generating different api_server addresses, we don't need a jinja-calculated environment variable.